website/blog/2023-01-18-release-0.11.md

```diff
@@ -0,0 +1,256 @@
+
+In some context, users need the ability to disable and re-enable the proxy configuration very quickly, without havint to entirely reconfigure it. This is now possible from the Podman Desktop settings page, where a toggle to enable/disable the proxy configuration has been added.
+
+<ReactPlayer playing controls url='https://user-images.githubusercontent.com/436777/205955418-670bc37c-a74f-40ef-bc60-8d9d013aa0dc.mp4' />
+
+Note: extensions can read this information and then update the proxy configuration.
+
+### Namespace selection for deploy to kubernetes [#1008](https://github.com/containers/podman-desktop/pull/1008) 
+
+A new dropdown menu has been added to the deploy to Kubernetes screen to allow selecting the namespace to deploy to. Defaults to defaultnamespace, if unable to get any from kube_client module.
+
+If the user have set `kubectl config set-context --current --namespace=NAMESPACE` it honors that as the chosen value.
+
+![deploy-kubernetes-namespace](https://user-images.githubusercontent.com/7339809/206688886-095e4f15-42ae-4a0a-b1c6-ae4b547fcdfb.gif)
+
+### Configure Podman binary path [#941](https://github.com/containers/podman-desktop/pull/941)
+
+Users may use a custom path to the podman binary. An option within settings (Settings -> Preferences) to allow providing an additional path to Podman binary has been added. This is useful for users who have installed Podman using a package manager and want to use the binary provided by the package manager. Additional information is available in the documentation for Mac users: [http://podman-desktop.io/docs/troubleshooting#unable-to-set-custom-binary-path-for-podman-on-macos](http://podman-desktop.io/docs/troubleshooting#unable-to-set-custom-binary-path-for-podman-on-macos)
+  
+![custom-path](https://user-images.githubusercontent.com/6422176/204832063-5858065a-2fc3-49de-8d23-3b99b7d10dbf.png)
+
+### Provide pre-defined registries [#1201](https://github.com/containers/podman-desktop/pull/1201)
+
+The experience of configuring a registry is getting simplified for the most popular ones. The user will be able to select a registry from a pre-defined list and will only have to provide the username and password. The list of pre-defined registries is the following:
+- Dockerhub
+- Red Hat Quay
+- GitHub
+- IBM Container Registry
+- Google Container Registry
+
+<ReactPlayer playing controls url='https://user-images.githubusercontent.com/6422176/213349477-979ee76f-1468-4663-8e49-c21cd6b276cf.mov' />
+
+### UI/UX Improvements 
+
+#### View Pods logs [#1122](https://github.com/containers/podman-desktop/pull/1122)
+
+The pods details view provides the ability to view the logs of each containers who may be running in a pod. Moreover, as it can also be hard to scan and identify from which container the outputs are coming, we appended the container name at the beginning of each output and differentiate each container with a different color. 
```

Comment:
"containers who" -> "containers that"
"from which container the outputs are coming" -> "which container is producing which output"?